app/person-escort-record/app/confirm/handover-details.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "form-wizard.njk" %}
+
+{% macro renderField(name) %}
+  {{ callAsMacro(options.fields[name].component)(options.fields[name]) }}
```

Comment:
I think this is at least our 2nd or 3rd instance of this function (there's something similar in population) - so next time we need this functionality, we should try to cleanup any tech debt